store/postgres/Cargo.toml

```diff
@@ -34,6 +34,9 @@ anyhow = "1.0.41"
 git-testament = "0.2.0"
 itertools = "0.10.1"
 
+# Our fork contains a small but hacky patch.
+web3 = { git = "https://github.com/graphprotocol/rust-web3", branch = "master" }
```

Comment:
I missed that the first time around - we try to import common dependencies from the `graph` crate so that we can update them in lockstep across crates. If you `use graph::prelude::web3::H256` etc. in your code, you don't need this additional dependency in the `store` crate.